### PR TITLE
[VideoVersion] Added VideoVersionPlay dialog

### PR DESCRIPTION
**Pull request by xodidox**
## Description

Added VideoVersionPlay dialog to give more flexibility to skinner
Added description for string to help translator
## Motivation and context

Based on feedback
## How has this been tested?

Runtime tested on MacBook
## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by ksooo** [comment]
Hi, @xodidox 

> Added description for string to help translator

That's not going to work like you did. Weblate does not support group comment for multiple strings. Translaters never look at the strings.po directly, they are using Weblate. The string you've added will be displayed in Weblate along with string `#40400`. You need to add a descriptive text (`.# `) for every single string.

> Added VideoVersionPlay dialog to give more flexibility to skinner

This dialog seems to be a dummy, without any functionality. What exactly can skinners make out of that? My skinning skills are really limited, so I'm just asking.

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -23918,7 +23918,7 @@ msgctxt "#40207"
 msgid "When enabled, video with multiple versions will be shown as folder in library, this folder can then be opened to display the individual video versions. When disabled, a video version dialog will be opened for the video."
 msgstr ""
 
-# Video versions
+#. System defined video version (for example, "Director's Cut") list
```
Comment (ksooo) [inline addons/resource.language.en_gb/resources/strings.po:23921]: This is not going to work. See my other comment.

**Comment by da-anda** [comment]
in order for your PRs to be easer to review it would be good if you could split up the changes into logical commits. Like in this PR one commit for the dialog and a second commit for the label descriptions. Since this PR is rather small, it might be a good exercise on the workflow for this, in case you are not that familiar with GIT yet.

**Comment by ksooo** [comment]
Yes, that would indeed be helpful, to back what @da-anda said.

**Comment by jjd-uk** [comment]
I've done a functional test, and it does exactly what me and @HitcherUK asked for. Thanks @xodidox

**Review by ksooo — APPROVED** [review]

**Review by Hitcher — APPROVED** [review]
Tested `videoversion` and `videoversionplay` windows and everything works as excepted.

I guess you can drop the label (id1) now as skins can use `Window.IsActive(videoversion)` or `Window.IsActive(videoversionplay)` for conditions.

Thanks.

**Review by jjd-uk — APPROVED** [review]

**Comment by xodidox** [comment]
> in order for your PRs to be easer to review it would be good if you could split up the changes into logical commits. Like in this PR one commit for the dialog and a second commit for the label descriptions. Since this PR is rather small, it might be a good exercise on the workflow for this, in case you are not that familiar with GIT yet.

Thanks for this tip. I'm not sure when and how, but I got the impression that each PR should only have one commit for Kodi repo. Maybe I was asked to squash commits? Don't remember. I'll adopt this in future PRs.

**Comment by jjd-uk** [comment]
For review it's easier for the changes to be in logical chunks. Once review is complete and PR is ready to be merged then it can be squashed down to one commit.

**Comment by ksooo** [comment]
The point is one commit per logical change , not one commit per PR. A PR can contain multiple logical changes, for example fix A and cleanup B. We want to have separate commits for those changes, one for the fix one for the cleanup.

**Pull request merged**